taxonomies/categories.txt

```diff
@@ -69098,6 +69098,19 @@ wikidata:en:Q531184
 
 <en:Meals with chicken
 en:Chicken Tikka
+hi:चिकन टिक्का, मसालेदार मुर्गा, तिक्के वाला मुर्गा
+fr:Poulet Tikka, Poulet mariné aux épices
```

Comment:
Synonyms must be specific to the taxonomy entry, here they are too generic and would match all other chicken with spices.